Apps/Medication/src/Controllers/PharmacyController.cs

```diff
@@ -59,28 +59,17 @@ public PharmacyController(IPharmacyService pharmacyService)
         public async Task<RequestResult<Pharmacy>> GetPharmacy(string pharmacyId)
         {
             HNMessage<Pharmacy> pharmacyResult = await this.pharmacyService.GetPharmacyAsync(pharmacyId).ConfigureAwait(true);
-
-            if (pharmacyResult.IsError)
+            RequestResult<Pharmacy> result = new RequestResult<Pharmacy>();
```

Comment:
nit: lets pass the parameters for initialization
```
RequestResult<Pharmacy> result = new RequestResult<Pharmacy>()
{
 ResultStatus = pharmacyResult.Result,
 ResultMessage = pharmacyResult.ResultMessage
}

Reply:
Fixed